src/app/db/schemas/ArticleSchema.ts

```diff
@@ -42,6 +42,9 @@ const ArticleSchema : Schema = new Schema({
 	toObject: {
 		retainKeyOrder: true,
 		transform: (doc : Document, converted : any) => {
+			// console.log('------------------------------------------------------------');
+			// console.log('ArticleSchema transform:', converted);
```

Comment:
These are my bad, I left them in for easy debugging of additional query functions 😊